bin/list

```diff
@@ -172,16 +172,25 @@ const handler = async (req, res) => {
 
   related = decodeURIComponent(path.format(related))
 
+  let notFoundResponse
+
+  try {
+    const custom404Path = path.join(current, '/404.html')
+    notFoundResponse = fs.readFileSync(custom404Path, 'utf-8')
+  } catch (err) {
+    notFoundResponse = 'Not Found'
```

Comment:
I also recommend making this `catch (err) {}` and set `'Not Found'` as the default value of `notFoundResponse` instead.